OrbitClientGgp/CMakeLists.txt

```diff
@@ -7,19 +7,18 @@ project(OrbitClientGgp)
 add_executable(OrbitClientGgp main.cpp)
 
 target_compile_options(OrbitClientGgp PRIVATE
-    ${STRICT_COMPILE_FLAGS})
```

Comment:
The CMakeLists.txt is not change at all now  so maybe it is not worth it to include these indentation updates in the commit

Reply:
The identation updates were not intentional but were caused by the formatting in CLion. However, I have found a way to remove the identation updates. Thanks you a lot for taking the time to review this and point this out!